Geometry/TrackerNumberingBuilder/src/GeometricDet.cc

```diff
@@ -216,19 +225,19 @@ namespace {
 }  // namespace
 
 void GeometricDet::deleteComponents() {
-  std::for_each(_container.begin(), _container.end(), Deleter());
-  _container.clear();
+  std::for_each(container_.begin(), container_.end(), Deleter());
+  container_.clear();
 }
 
 GeometricDet::Position GeometricDet::positionBounds() const {
-  Position _pos(float(_trans.x() / cm), float(_trans.y() / cm), float(_trans.z() / cm));
-  return _pos;
+  Position pos_(float(trans_.x() / cm), float(trans_.y() / cm), float(trans_.z() / cm));
```

Comment:
convertMmToCm?

Reply:
yes  basically just changed all GeometricDet data members from _member to member_, so did not author this. But yes you are right, will use geant_units::operators::convertMmToCm